.github/workflows/auto_cherry_pick.yml

```diff
@@ -6,20 +6,15 @@ on:
     types:
       - closed
 
+# Github & Parent PR Env vars
+env:
+  assignee: ${{ github.event.pull_request.assignee.login }}
+  title: ${{ github.event.pull_request.title }}
+  number: ${{ github.event.number }}
+
 jobs:
-  previous-branch:
-    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'CherryPick')
-    name: Calculate previous branch name
-    runs-on: ubuntu-latest
-    outputs:
-      previous_branch: ${{ steps.set-branch.outputs.previous_branch }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - id: set-branch
-        run: echo "previous_branch=$(if [ $GITHUB_BASE_REF  == 'master' ]; then echo $(git branch -rl 'origin/6.*.z' --format='%(refname:lstrip=-1)' | sort --version-sort | tail -n1); else echo '6.'$(($(echo $GITHUB_BASE_REF | cut -d. -f2) - 1))'.z'; fi)" >> $GITHUB_OUTPUT
 
+  # Pre-Requisites for Auto Cherrypicking
```

Comment:
I don't think so it is prerequisite for auto cherrypicking, we could say may be auto merging auto-cherrypick PR's

Reply:
Here I mean all the pre-requisites needed for the entire auto-cherry-pick job.